### PR TITLE
Document issue management workflow

### LINKED ISSUE #2032: Styleguide for issue tagging
I liked @lresende's idea of having a guide documenting our issue tags. Maybe we can put together a markdown table similar to this, but more specific to us?

|  | Tags |
| --- | --- |
| Platform | ![][angular] ![][node] |
| Problems | ![][bug] ![][security] ![][production] |
| Mindless | ![][chore] ![][legal] |
| Experience | ![][copy] ![][design] ![][ux] |
| Environment | ![][staging] ![][test] |
| Feedback | ![][discussion] ![][rfc] ![][question] |
| Improvements | ![][enhancement] ![][optimization] |
| Additions | ![][feature] |
| Pending | ![][in progress] ![][watchlist] |
| Inactive | ![][invalid] ![][wontfix] ![][duplicate] ![][on hold] |

[angular]: https://img.shields.io/badge/-angular-99ccff?style=flat
[node]: https://img.shields.io/badge/-node-99ccff?style=flat
[bug]: https://img.shields.io/badge/-bug-red?style=flat
[security]: https://img.shields.io/badge/-security-red?style=flat
[production]: https://img.shields.io/badge/-production-red?style=flat
[chore]: https://img.shields.io/badge/-chore-yellow?style=flat
[legal]: https://img.shields.io/badge/-legal-yellow?style=flat
[copy]: https://img.shields.io/badge/-copy-orange?style=flat
[design]: https://img.shields.io/badge/-design-orange?style=flat
[ux]: https://img.shields.io/badge/-ux-orange?style=flat
[staging]: https://img.shields.io/badge/-staging-pink?style=flat
[test]: https://img.shields.io/badge/-test-pink?style=flat
[discussion]: https://img.shields.io/badge/-discussion-purple?style=flat
[rfc]: https://img.shields.io/badge/-rfc-purple?style=flat
[question]: https://img.shields.io/badge/-question-purple?style=flat
[enhancement]: https://img.shields.io/badge/-enhancement-blue?style=flat
[optimization]: https://img.shields.io/badge/-optimization-blue?style=flat
[feature]: https://img.shields.io/badge/-feature-green?style=flat
[in progress]: https://img.shields.io/badge/-in_progress-yellow?style=flat
[watchlist]: https://img.shields.io/badge/-watchlist-yellow?style=flat
[invalid]: https://img.shields.io/badge/-invalid-grey?style=flat
[wontfix]: https://img.shields.io/badge/-wontfix-grey?style=flat
[duplicate]: https://img.shields.io/badge/-duplicate-grey?style=flat
[on hold]: https://img.shields.io/badge/-on_hold-grey?style=flat

### PR DESCRIPTION
**Pull request by lresende**
Fixes #2032 
 
Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

**Review by karlaspuldaro — COMMENTED** [review]
A few comments/questions
- what exactly `copy` tag in `experience` would be used for?
- `rfc` tag is not very clear as an abbreviation
- I'd change `mindless` category for `task` or something like that?
- I'd also use `program` (or something related) instead of `mentoring`
- Not sure why but the tag style for area/status/impact are not being rendered:
image

File: docs/source/developer_guide/issue-management.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+|  | Tags |
+| --- | --- |
+| Problems | ![][bug] ![][security] ![][production] |
+| Experience | ![][copy] ![][design] ![][ux] |
+| Feedback | ![][discussion] ![][rfc] ![][question] |
+| Improvements | ![][enhancement] ![][optimization] |
+| Area | ![][backend] ![][community] ![][documentation] ![][front-end] |
+| Satus | ![][needs discussion] ![][needs review]  ![][needs triage] ![][needs update] ![][waiting for author] ![][work in progress]|
+| Impact | ![][blocker] ![][bocked]  ![][do not merge]|
```
Comment (bourdakos1) [inline docs/source/developer_guide/issue-management.md:33]:
```suggestion
| Status | ![][needs discussion] ![][needs review]  ![][needs triage] ![][needs update] ![][waiting for author] ![][work in progress] |
| Impact | ![][blocker] ![][bocked]  ![][do not merge] |
```

Comment (kevin-bates) [reply docs/source/developer_guide/issue-management.md:33]: There's also a typo in `Impact`: `bocked` -> `blocked`

**Comment by ptitzler** [comment]
- need to add the new doc to the index for this to show up in the TOC
- should define the tags themselves since it's not necessarily clear what they mean. for example, what's the difference between content parser and content service.

File: docs/source/developer_guide/issue-management.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+[front-end]: https://img.shields.io/badge/-front_end-c5def5?style=flat
+<!--Impact-->
+[blocked]: https://img.shields.io/badge/-blocked-d93f0b?style=flat
+[blocker]: https://img.shields.io/badge/-blocker-d93f0b?style=flat
+[do not merge]: https://img.shields.io/badge/-do_not_merge-d93f0b?style=flat
+[needs doc updates]: https://img.shields.io/badge/-needs_doc_updates-d93f0b?style=flat
+[needs example updates]: https://img.shields.io/badge/-needs_example_updates-d93f0b?style=flat
+[needs release note]: https://img.shields.io/badge/-needs_release_note-d93f0b?style=flat
+<!--Help Wanted-->
+[good first use]: https://img.shields.io/badge/-good_first_use-84e251?style=flat
```
Comment (bourdakos1) [inline docs/source/developer_guide/issue-management.md:58]:
```suggestion
[good first use]: https://img.shields.io/badge/-good_first_issue-84e251?style=flat
```

File: docs/source/developer_guide/issue-management.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+
+### Labels used for tagging Issues and Pull Requests
+
+| Keyword | Tags |
+| --- | --- |
+| Area | ![][backend] ![][community] ![][documentation] ![][front-end] |
+| Impact | ![][blocker] ![][blocked]  ![][do not merge] ![][needs doc updates] ![][needs example updates] ![][needs release note] |
+| Help Wanted | ![][good first use] ![][good first project] ![][help wanted] |
+| Mentoring Program | ![][jumpstart] ![][rcos] |
+| Kind | ![][accessibility] ![][accessibility] ![][bug] ![][config] ![][enhancement] ![][investigate] ![][no functionality change] ![][question] ![][task] ![][user error] |
```
Comment (bourdakos1) [inline docs/source/developer_guide/issue-management.md:30]: duplicate
```suggestion
| Kind | ![][accessibility] ![][bug] ![][config] ![][enhancement] ![][investigate] ![][no functionality change] ![][question] ![][task] ![][user error] |
```

File: docs/source/developer_guide/issue-management.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+| Feedback | ![][isv] ![][proposal] ![][user] |
+| External | ![][jupyter enterprise gateway] ![][jupyter server] ![][jupyter hub] ![][jupyter lab 1.x] ![][jupyter lab 2.x] ![][jupyter lab 3.x] ![][jupyter lab 4.x] ![][upstream] |
+| Component | ![][application utils] ![][binder] ![][build] ![][cli tools] ![][code snippets] ![][component registry] ![][content parser] ![][docker] ![][git] ![][install] ![][lsp] ![][metadata editor] ![][metadata] ![][performance] ![][pipeline editor visual studio] ![][pipeline editor] ![][pipeline runtime] ![][pipeline validation] ![][python editor] ![][r editor] ![][resource usage] ![][s3] ![][submit notebook button] ![][submit script button] ![][test] ![][toc] ![][troubleshooting] |
+| Platform | ![][open data hub] ![][OpenShift] ![][Pipeline Airflow] ![][Pipeline Kubeflow] ![][Pipeline Local] ![][Windows] |
+| Browser | ![][chrome] ![][firefox] ![][safari] |
+| Inactive | ![][cannot reproduce] ![][deferred] ![][duplicate] ![][invalid] ![][obsolete] ![][wontfix] |
+| Satus | ![][needs discussion] ![][needs triage] ![][needs update] ![][waiting for author] ![][work in progress] |
```
Comment (bourdakos1) [inline docs/source/developer_guide/issue-management.md:37]:
```suggestion
| Status | ![][needs discussion] ![][needs triage] ![][needs update] ![][waiting for author] ![][work in progress] |
```

**Review by bourdakos1 — CHANGES_REQUESTED** [review]

**Review by bourdakos1 — APPROVED** [review]

**Pull request merged**